src/components/PrimaryNav/_primary-nav.scss

```diff
@@ -0,0 +1,57 @@
+@import "vanilla-framework/scss/settings_colors";
+
+.p-primary-nav {
+  a,
+  a:hover {
+    color: $color-light;
+  }
+
+  &__header {
+    display: grid;
+    grid-template-columns: 1fr 40px;
+    margin-bottom: -1rem;
+    padding: 0.5rem;
+  }
+
+  &__toggle {
+    background: none;
+    border: 0;
+    transition: transform 0.25s ease-in-out;
+
+    &:hover {
+      background: 0;
+    }
+  }
+
+  .p-list__link {
+    color: $color-light;
+  }
+
+  .p-list__item {
+    padding-left: 2.5rem;
+
+    &.is-selected {
+      background-color: #044f66;
+      border-left: 3px solid #3fc8f2;
+      margin-right: -0.5rem;
+      padding-left: calc(2.5rem - 3px);
```

Comment:
If all links have a transparent 3px border on the left then selected you only need to change the colour. Saves complexity and repaints

Reply:
Didn't think of that - nice.

Reply:
Hah, smart :D